File: cli-tools/automation/node/get-webhook-public-key.js
const axios = require('axios');
const { BASE_URL, getAccessToken } = require('./utils');

async function getWebhookPublicKey() {
    const token = await getAccessToken();
    try {
        const response = await axios.get(`${BASE_URL}/webhook/public-keys`, {
            headers: { 'Authorization': `Bearer ${token}` }
        });
        console.log(JSON.stringify(response.data, null, 2));
    } catch (error) {
        console.error('Error:', error.response ? error.response.data : error.message);
    }
}

getWebhookPublicKey();

File: cli-tools/automation/python/get_webhook_public_key.py
import json
import requests
from utils import BASE_URL, get_access_token

def get_webhook_public_key():
    token = get_access_token()
    headers = {'Authorization': f'Bearer {token}'}
    
    try:
        response = requests.get(f"{BASE_URL}/webhook/public-keys", headers=headers)
        response.raise_for_status()
        print(json.dumps(response.json(), indent=2))
    except Exception as e:
        print(f"Error: {e}")

if __name__ == "__main__":
    get_webhook_public_key()
